### DISCUSSION TITLE
Is it possible to generate Class Activation Map (CAM) and Gradient-CAM (GRAD-CAM)?

### DISCUSSION THREAD

**Discussion by yohanssen** [timeseriesAI/tsai#696] [Q&A]
Hello, is it possible in Tsai to generate Class Activation Map (CAM) and Gradient-CAM (GRAD-CAM) like in the timeseries fast-ai (https://ai-fast-track.github.io/timeseries/univariate_timeseries_CAM/)? thank you

**Comment by oguiza** [comment]
Hi @yohanssen,
No, it's not possible out of the box.
Would you be interested in creating a PR?

**Reply by yohanssen** [reply]
Thank you for your reply.  Currently, I could use the  inception_time model from timeseries to generate CAM.  I will try to see possibilities to adopt the 'timeseries cam' with other model from Tsai.
